bentoml/cli/azure_function.py

```diff
@@ -0,0 +1,423 @@
+    )
+    @click.option(
+        '--premium-plan-sku',
+        type=click.STRING,
+        default=DEFAULT_PREMIUM_PLAN_SKU,
+        help='The Azure premium SKU for the deployment. Options are EP1, EP2, and EP3.'
+        ' The default value is EP1',
+    )
+    @click.option(
+        '-l',
+        '--labels',
+        type=click.STRING,
+        callback=parse_labels_callback,
+        help='Key:value pairs that are attached to deployments and intended to be used '
+        'to specify identifying attributes of the deployments that are meaningful to '
+        'users. Multiple labels are separated with `,`',
+    )
+    @click.option(
+        '--function-auth-level',
+        type=click.STRING,
+        default=DEFAULT_FUNCTION_AUTH_LEVEL,
+        help='The authorization level for the deployed Azure function. '
+        'Options are "anonymous", "function", and "admin". Default value is '
+        'function',
+    )
+    @click.option('-o', '--output', type=click.Choice(['json', 'yaml']), default='json')
+    @click.option(
+        '--wait/--no-wait',
+        default=True,
+        help='Wait for apply action to complete or encounter an error.'
+        'If set to no-wait, CLI will return immediately. The default value is wait',
+    )
+    def deploy(
+        namespace,
+        name,
+        bento,
+        location,
+        min_instances,
+        max_burst,
+        premium_plan_sku,
+        labels,
+        function_auth_level,
+        output,
+        wait,
+    ):
+        track_cli('deploy-create', PLATFORM_NAME)
+        bento_name, bento_version = bento.split(':')
+        yatai_client = YataiClient()
+        try:
+            with Spinner('Deploying Azure function deployment'):
+                result = yatai_client.deployment.create_azure_function_deployment(
+                    name=name,
+                    namespace=namespace,
+                    labels=labels,
+                    bento_name=bento_name,
+                    bento_version=bento_version,
+                    location=location,
+                    min_instances=min_instances,
+                    max_burst=max_burst,
+                    premium_plan_sku=premium_plan_sku,
+                    function_auth_level=function_auth_level,
+                    wait=wait,
+                )
+                if result.status.status_code != status_pb2.Status.OK:
+                    error_code, error_message = status_pb_to_error_code_and_message(
+                        result.status
+                    )
+                    _echo(
+                        f'Failed to create Azure function deployment {name} '
+                        f'{error_code}:{error_message}',
+                        CLI_COLOR_ERROR,
+                    )
+                    return
+                track_cli('deploy-create-success', PLATFORM_NAME)
+                _echo(
+                    f'Successfully created Azure function deployment {name}',
```

Comment:
s/"Azure Functions"/"Azure function" - make sure to also change other places

Reply:
Just realized that you were using "azure function" in the code .. the actual service name is "azure functions" I believe